sklearn/svm/tests/test_svm.py

```diff
@@ -1519,3 +1519,13 @@ def test_n_iter_libsvm(estimator, expected_n_iter_type, dataset):
     if estimator in [svm.SVC, svm.NuSVC]:
         n_classes = len(np.unique(y))
         assert n_iter.shape == (n_classes * (n_classes - 1) // 2,)
+
+
+@pytest.mark.parametrize("Klass", [SVR, NuSVR, OneClassSVM])
```

Comment:
I'm trying to push for these "TODO(version_number)" so they are easier to grep for.

```suggestion
# TODO(1.3) Remove
@pytest.mark.parametrize("Klass", [SVR, NuSVR, OneClassSVM])
```